### PR TITLE
Drawer and other minor changes

### LINKED ISSUE #70: Clickable trip points
As user
I want click on the trophy icon to see what the points mean

Requirements:

- A pop up info with explanatory content is added on the trophy button tap

### LINKED ISSUE #75: Navigation icons update
As user
I want to have a pretty app experience.

Requirements:

- line icons pack is searched for icons that could better fit the bottom tab

### LINKED ISSUE #76: Hamburger
As user
I want to have a hamburger button in the app

Requirements:

- Hamburger icon is added
- A drawer is created with a logo, app version and copyright info for a start

### LINKED ISSUE #77: Info page title overlay
As user
I want to see no overlay when I scroll down info page

Requirements:

Info page is redesigned so that the app bar does not overlay the content when scrolling

### PR DESCRIPTION
**Pull request by knutm92**
fixes #76 
fixes #77 
fixes #70 
fixes #75

### CODE REVIEW

File: lib/widgets/infoView_widget.dart [hunk 1]
```diff
@@ -19,245 +19,283 @@ class InfoView extends StatelessWidget {
...
+                          text: "Twórcy aplikacji",
+                        ),
+                        subtitle: Text("Kim jesteśmy?"),
+                        children: [
+                          Container(
+                            margin: EdgeInsets.symmetric(
+                                horizontal: Constants.insideMargin),
+                            child: ClipRRect(
+                              child: Image.asset("assets/images/gms_team_white.png"),
+                              borderRadius: BorderRadius.all(Radius.circular(20)),
```
Comment (darowsluk) [inline lib/widgets/infoView_widget.dart:277]: Consider using a constant.

**Review by darowsluk — CHANGES_REQUESTED** [review]
Great changes! Especially the drawer makes a difference together with the image. :)

**Comment by darowsluk** [comment]
@knutm92 Sorry, I let myself do a rebase from dev to pickup the latest build changes. This allows for automatic build verification, which reported all checks have passed.

File: lib/view/mapCard_screen.dart [hunk 2]
```diff
@@ -22,33 +22,6 @@ class MapCardPage extends StatelessWidget {
     return Scaffold(
       extendBodyBehindAppBar: true,
       backgroundColor: AppColors.primaryWhite,
-      appBar: AppBar(
```
[the remark this answers is not in the document]
Comment (knutm92) [reply lib/view/mapCard_screen.dart:25]: > @knutm92 Sorry, I let myself do a rebase from dev to pickup the latest build changes. This allows for automatic build verification, which reported all checks have passed.

Thank you, this is actually good as I could jump straight into resolving suggestions, also not feeling in top form after the sickness.

File: lib/view/home_screen.dart [hunk 3]
```diff
@@ -56,41 +59,25 @@ class _HomePageState extends State<HomePage> {
...
-                  ),
-                  backgroundColor: AppColors.primaryNormal,
-                ));
-              },
-            )
-          ],
         ),
         bottomNavigationBar: TabBar(
-          labelColor: AppColors.primaryWhite,
+          labelColor: Color.fromRGBO(238, 241, 241, 1),
```
[the remark this answers is not in the document]
Comment (knutm92) [reply lib/view/home_screen.dart]: I think this unwanted change sneaked in when I tried to get the color description for our photo editor

File: lib/view/home_screen.dart [hunk 4]
```diff
@@ -56,41 +59,25 @@ class _HomePageState extends State<HomePage> {
...
           unselectedLabelColor: AppColors.primaryNormal,
           indicator: TabOutlineIndicator(),
           tabs: [
             Tab(
                 icon: Icon(
-              LineIcons.tripadvisor,
+              LineIcons.walking,
```
[the remark this answers is not in the document]
Comment (knutm92) [reply lib/view/home_screen.dart]: Yep, compass icon is more in line with the app design on the whole :)

File: lib/view/trip_screen.dart [hunk 5]
```diff
@@ -42,30 +42,43 @@ class TripPage extends StatelessWidget {
...
-                    padding: EdgeInsets.all(12),
-                    decoration: BoxDecoration(
-                      color: AppColors.primaryNormal
-                          .withOpacity(Constants.opacity75),
-                      shape: BoxShape.rectangle,
-                      borderRadius: BorderRadius.all(
-                        Radius.circular(Constants.borderRadius),
+                ? GestureDetector(
+                    onTap: () {
+                      ScaffoldMessenger.of(context).showSnackBar(const SnackBar(
```
[the remark this answers is not in the document]
Comment (knutm92) [reply lib/view/trip_screen.dart:48]: Good suggestion to add the `removeCurrentSnackBar()` I missed that. 

Regarding creating `ScaffoldMessanger` custom class- I'm not sure about it, fixed parameters copied from `ScaffoldMessanger` in `cardHero_widget.dart` didn't fit the `trip_screen` either throwing an error or resulting in button overlay. I'm wondering whether there is  a cleaner way to keep class parameters fixed/constrained to a selected group in dart other than to just use class wrappers, maybe we could discuss this on some occasion.

File: lib/widgets/drawerWidget.dart [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+              child: Column(
+                children: [
+                  const SizedBox(height: Constants.bottomMargin),
+                  Image.asset(
+                      "assets/images/icons/wsd_logo_white_transparent.png",
+                      scale: Constants.drawerLogoScale,
+                      fit: BoxFit.fitWidth),
+                  const SizedBox(height: Constants.bottomMargin),
+                  KompasText(
+                      style: AppTextStyles.headerH3White, text: "kompas.sds"),
```
[the remark this answers is not in the document]
Comment (knutm92) [reply lib/widgets/drawerWidget.dart]: Good point

File: lib/widgets/drawerWidget.dart [hunk 7]
```diff
@@ -0,0 +1,96 @@
...
+                          TextSpan(
+                            text: 'www.wsd.sds.pl',
+                            style: TextStyle(
+                                color: AppColors.primaryWhite,
+                                decoration: TextDecoration.underline),
+                            recognizer: TapGestureRecognizer()
+                              ..onTap = () {
+                                launch('https://wsd.sds.pl/');
+                              },
+                          ),
```
[the remark this answers is not in the document]
Comment (knutm92) [reply lib/widgets/drawerWidget.dart:73]: I added that and reduced the font a bit as well

**Comment by knutm92** [comment]
@darowsluk Thanks for the review and rebase, I have updated the issues descriptions and pushed changes suggested in the review!

File: lib/view/trip_screen.dart (hunk 5 above)
[the remark this answers is not in the document]
Comment (darowsluk) [reply lib/view/trip_screen.dart:48]: Ok. We can create a separate issue for ScaffoldMessenger.

File: lib/widgets/drawerWidget.dart (hunk 7 above)
[the remark this answers is not in the document]
Comment (darowsluk) [reply lib/widgets/drawerWidget.dart:73]: Looks good!

**Review by darowsluk — APPROVED** [review]
Everything looks good. Let's merge.

**Pull request merged**